### ISSUE TITLE
 ERROR in ./mocha-env-loader.js!./example/test/index.js Module build failed: TypeError: undefined is not a function

### ISSUE THREAD

**Issue by claudiopro** [webpack-contrib/karma-webpack#80]
## Steps to reproduce:
1. Run `npm install babel-loader --save-dev`
2. Add the `babel` loader to `karma-complex.conf.js`[1]
3. Run `./node_modules/karma/bin/karma start example/karma-complex.conf.js --single-run`

[1]

``` js
devtool: 'inline-source-map',
resolve: {
  extensions: ["", ".js", ".jsx", ".coffee"]
},
module: {
  loaders: [
    { test: /\.coffee$/, loader: "coffee-loader" },
    {
      test: /\.jsx?$/,
      exclude: /(node_modules|bower_components)/,
      loader: 'babel-loader'
    }
  ]
}
```
## Issue:

```
    ERROR in ./mocha-env-loader.js!./example/test/index.js
    Module build failed: TypeError: undefined is not a function
      at Object.module.exports (/Users/delphine/Development/Git/claudiopro/karma-webpack/mocha-env-loader.js:12:31)

    ERROR in ./mocha-env-loader.js!./example/test/separate.js
    Module build failed: TypeError: undefined is not a function
      at Object.module.exports (/Users/delphine/Development/Git/claudiopro/karma-webpack/mocha-env-loader.js:12:31)
```

Line `mocha-env-loader.js:12:31` refers to a non-existing method `SourceNode.fromSourceWithMap()`:

``` js
var SourceNode = require("source-map").SourceNode;
module.exports = function(content, map) {
//...
    if(map) {
        var sourceNode = SourceNode.fromSourceWithMap(content, map);
    } else {
//...
};
```

https://github.com/mozilla/source-map offers a `SourceNode.fromStringWithSourceMap()` method instead.

**Comment by zpratt** [comment]
@claudiopro I've noticed this in all versions of this package after 1.5.1. For now, you can try strictly using that version as a temporary workaround.

**Comment by claudiopro** [comment]
@zpratt thank you, will stick to v1.5.1

**Comment by joaorafaelm** [comment]
Hello, I am currently using v1.51 and the error still persists, the tests run with no problems, but is there a way I can supress these errors from console?

**Comment by tschaub** [comment]
In case there is anyone still watching this, it should be addressed by #238.
